website/project/views/node.py

```diff
@@ -952,6 +952,8 @@ def get_summary(**kwargs):
     auth = kwargs['auth']
     node = kwargs['node'] or kwargs['project']
     rescale_ratio = kwargs.get('rescale_ratio')
+    if rescale_ratio is None and request.args.get('rescale_ratio'):
+        rescale_ratio = float(request.args.get('rescale_ratio'))
```

Comment:
The possible `TypeError` and `ValueError` that could occur here is still unhandled. If the `rescale_ratio` argument is not valid (i.e. not a number), an error response should be returned to the client by raising an `HTTPError`.